CMakeLists.txt

```diff
@@ -5,23 +5,30 @@ project(detray VERSION 0.1 LANGUAGES CXX)
 option(CMAKE_WARN_DEPRECATED "Not warning for deprecation in cmake" Off)
 enable_testing()
 
-option(DETRAY_ARRAY_PLUGIN "Build std::array math plugin" On)
+option(DETRAY_ARRAY_PLUGIN "Build std::array math plugin" Off)
```

Comment:
Do you want to have this off by default?

Reply:
Otherwise this can go in as is.

Reply:
It is more of a leftover, to be honest. This would mean there is no automatic performance plot when a commit is pushed?